Makefile

```diff
@@ -58,6 +58,8 @@ endif
 ci: install-tools generate test coverage
 
 $(BINARY):
+	CGO_ENABLED=1 \
+	CGO_CFLAGS="-O2 -D__BLST_PORTABLE__" \
```

Comment:
this be ok with quotes?

Reply:
Yeah... this one is OK lol.  Just to make things more confusing 🙃